story/story_manager.py

```diff
@@ -267,6 +267,8 @@ def story_context(self):
 
 class UnconstrainedStoryManager(StoryManager):
     def act(self, action_choice):
+        if self.generator.raw:
+            action_choice = " " + action_choice + " "

```

Comment:
Do you think this is adequate? I didn't want to force newlines since you might just be supplying some intermediate sentence in a paragraph, and I added the ability to write '\n' for a newline in your input explicitly.

Reply:
You may want to skip adding the space if `action_choice` already ends with one, or ends with a `\n`.

Reply:
Okay, now if the user inputs some other whitespace, that will be used instead.